Makefile

```diff
@@ -24,39 +24,40 @@ init:
 	@echo "'make init' is no longer needed"
 
 clean:
-	$(RM) -rf build docs/_build docs/api htmlcov
+	$(RM) -rf build docs/_build docs/api htmlcov ctapipe.egg-info dist
 	find . -name "*.pyc" -exec rm {} \;
 	find . -name "*.so" -exec rm {} \;
 	find . -name __pycache__ | xargs rm -fr
 
 test:
-	$(PYTHON) setup.py test
+	pytest
 
 doc:
-	cd docs && $(MAKE) html
+	cd docs && $(MAKE) html SPHINXOPTS="-W --keep-going -n --color -j auto"
 	@echo "------------------------------------------------"
 	@echo "Documentation is in: docs/_build/html/index.html"
 
 doc-publish:
 	ghp-import -n -p -m 'Update gh-pages docs' docs/_build/html
 
-conda:
-	python setup.py bdist_conda
-
 analyze:
 	@pylint ctapipe --ignored-classes=astropy.units
 
-pep8:
-	@pep8 --statistics
+flake8:
```

Comment:
In addition to the above we can change the CI (and here) to e.g. `make lint` or similar?

Reply:
i like that